sdks/python/apache_beam/runners/direct/executor.py

```diff
@@ -292,43 +297,66 @@ def call(self):
     side_input_values = [self._side_input_values[side_input]
                          for side_input in self._applied_ptransform.side_inputs]
 
-    try:
-      evaluator = self._transform_evaluator_registry.get_evaluator(
-          self._applied_ptransform, self._input_bundle,
-          side_input_values, scoped_metrics_container)
-
-      if self._fired_timers:
-        for timer_firing in self._fired_timers:
-          evaluator.process_timer_wrapper(timer_firing)
-
-      if self._input_bundle:
-        for value in self._input_bundle.get_elements_iterable():
-          evaluator.process_element(value)
-
-      with scoped_metrics_container:
-        result = evaluator.finish_bundle()
-        result.logical_metric_updates = metrics_container.get_cumulative()
+    # Switch to turn on/off the retry of bundles.
```

Comment:
<!--thread_id:cc_132461533_t; commit:e722041a1a4f5ab47ea37928097194c25a8cb831; resolved:0-->
<!--section:context-quote-->
> **mariapython** wrote:
> Done.

<!--section:body-->
This is still not addressed at the latest commit. You can move this entire block to the `__init__` constructor.  Can you please fix?